CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [develop]
 
+## Changed
+- Update to Bitmovin Player v8
+- Switch to Webpack and yarn
```

Comment:
Which team is responsible for this project now? If it's still the player team, please revert to `npm`. We are not using `yarn` as there is no functional advantage over recent `npm` versions.